core/theme/classic.js

```diff
@@ -10,6 +10,11 @@
  */
```

Comment:
Do we need to add any aliases for this and other files in core/theme?

Reply:
Yeah, there are missing JSDoc for the themes here.
I'll address this in a follow up PR.